buildenv/jenkins/JenkinsfileBase

```diff
@@ -144,52 +144,33 @@ def setupParallelEnv() {
 					childTest = "test_iteration_${i}"
 					buildListName = env.BUILD_LIST
 				}
+
+				def childParams = []
+				// loop through all the params and change the parameters if needed
+				params.each { param ->
+					if (param.key == "ITERATIONS") {
+						childParams << string(name: param.key, value: "1")
+					} else if (param.key == "BUILD_LIST") {
+						childParams << string(name: param.key, value: "${buildListName}")
+					} else if (param.key == "IS_PARALLEL") {
+						childParams << booleanParam(name: param.key, value: false)
+					} else {
+						def value = param.value.toString()
+						if (value == "true" || value == "false") {
```

Comment:
I guess this works because we either have Boolean parameters or String parameters (as our Choice/Credential parameters end up being handled as Strings).  :)

Reply:
yes, choice/credential parameters end up being handled as Strings.